src/controllers/usersurveyresponse.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { getRepository } from 'typeorm';
+import { NextFunction, Request, Response } from 'express';
+
+import UserSurveyResponse from '../entity/UserSurveyResponse';
+import Answer from '../entity/Answer';
+import Choice from '../entity/Choice';
+
+export const createSurveyResponse = async (
+    req: Request,
+    res: Response
+): Promise<Response> => {
+    const responseFromReq = {
+        userId: req.body.user_id,
+        surveyId: req.body.survey_id,
```

Comment:
Tutaj chyba cammelCase będzie. Chociaz to zalezy od Konrada bo z tego co mam zapisane to on robi widok do wypełniania ankiety 
```suggestion
        userId: req.body.userId,
        surveyId: req.body.surveyIId,
```